pyproject.toml

```diff
@@ -36,6 +36,11 @@ doc = [
     "Sphinx==5.3.0",
     "sphinx-copybutton==0.5.1",
     "sphinx-notfound-page==0.8.3",
+    "sphinx-design",
```

Comment:
Let us fix the version for `sphinx-design` too. I know this implies downloading the examples from the right release branch, so let us fix that too.